vue/components/ui/animations/animation-visualizer/animation-visualizer.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div class="animation-visualizer">
+        <h3 v-if="title">{{ title }}</h3>
+        <div class="box" v-bind:style="animationStyle" v-if="renderComponent">
+            Box
+        </div>
+    </div>
+</template>
+
+<style lang="scss">
+@import "css/animations.scss";

```

Comment:
Maybe it would look better with this:
`.animation-visualizer {
    padding: 10px;
    text-align: center;
    display: inline-block;
    width: 200px;
    margin: 5px;
}`